### PR TITLE
copy and paste settings

### PR DESCRIPTION
**Pull request by jh-bate**
@krystophv still a WIP but if you get the chance to have a look that would be appreciated

### CODE REVIEW

**Comment by krystophv** [comment]
[It's hard to copy/paste Device Settings into an email AND edit them (i.e. add a new basal segment)](https://trello.com/c/ICFnscSF/115-it-s-hard-to-copy-paste-device-settings-into-an-email-and-edit-them-i-e-add-a-new-basal-segment)

**Comment by krystophv** [comment]
LGTM

File: test/components/settings/NonTandem.test.js [hunk 1]
```diff
@@ -1,546 +0,0 @@
-/*
```
Comment (krystophv) [inline test/components/settings/NonTandem.test.js]: Was this file deleted unintentionally? Looks like it was part of a temporary test removal and didn't get restored as part of the undo.

File: src/utils/format.js [hunk 2]
```diff
@@ -65,3 +66,46 @@ export function displayBgValue(val, units, outOfRangeThresholds) {
   }
   return format('d')(val);
 }
+
+/**
+ * patientFullName
+ * @param  {Object} patient   the patient object that contains the profile
+ * @return {String}           fullName of patient or empty if there is none
+ */
+export function patientFullName(patient) {
```
Comment (krystophv) [inline src/utils/format.js]: Couldn't find any tests for these new utility functions.

**Comment by krystophv** [comment]
LGTM

**Comment by jebeck** [comment]
OK @jh-bate this is updated from master with my recent refactoring + node update. I didn't make a new tag since it sounded like you had another substantive change to make on this branch before it goes back out for testing.

**Pull request merged**